api-client/package.json

```diff
@@ -29,7 +29,6 @@
     "@types/jest": "^26.0.23",
     "@types/js-md5": "^0.4.2",
     "@types/lodash": "^4.14.168",
-    "@types/node": "^15.0.2",
```

Comment:
Are you sure this is not used in jest tests? Maybe it makes sense to keep these types as we can call node functions in jest tests.

Reply:
These are installed globally for the whole project so they are redundant here.